app/views/mailers/notify_mailer/tag_moderator_confirmation_email.text.erb

```diff
@@ -1,14 +1,14 @@
 Hey <%= @user.name %>!
 
-Thanks for volunteering to be a <%= community_name %> tag moderator.  We’ve added you as a moderator for https://dev.to/t/<%= @tag %>. You’re currently the only non-core <%= community_name %> member that’s moderating this tag. Over time, we’ll add additional mods to join you because we view moderation as a collaborative endeavor that benefits from the support of multiple people.
+Thanks for volunteering to be a <%= community_name %> tag moderator.  We’ve added you as a moderator for <%= tag_url(@tag) %>. You’re currently the only non-core <%= community_name %> member that’s moderating this tag. Over time, we’ll add additional mods to join you because we view moderation as a collaborative endeavor that benefits from the support of multiple people.
```

Comment:
Here `tag_url` is used, which will use domain from action mailer `smtp_settings` (`ENV["APP_DOMAIN"]`) which could be fine, but in other places `app_url` is used. I suppose we should be consistent with the urls in email templates.

Reply:
@lightalloy I'm not quite sure what you mean, but I'll take a stab at a response. When an application helper like tag_url, comment_url, article_url, or user_url is more apt I use that to substitute the hardcoded url. But if we cannot use one of the more specific helpers, I then use app_url. This is true for the most part but I'm sure that there is one or two places where I need to clean up.